src/test/java/com/epam/steps/admin/CreateSubjectSteps.java

```diff
@@ -0,0 +1,158 @@
+        logger.info("Check subject is not added in the DB");
+        assertThat(dbHelper.isSubjectAddedInTheDB())
+                .withFailMessage("Subject wasn't meant to be added in the DB, but was added.")
+                .isTrue();
+    }
+
+    @Then("Check subject is added in the DB")
+    public void checkSubjectIsAddedInTheDB() {
+        logger.info("Check subject is added in the DB");
+        assertThat(dbHelper.isSubjectAddedInTheDB())
+                .withFailMessage("Subject was meant to be added in the DB, but wasn't added.")
+                .isFalse();
+    }
+
+    @Then("Check subject is displayed in the list")
+    public void checkSubjectIsDisplayedInTheList() {
+        assertThat(adminPage.checkNewCreatedItemIsDisplayedOnAdminsSection())
+                .withFailMessage("Subject is not displayed in the list, but it should be")
+                .isTrue();
+    }
+
+    @Then("Check error message of existed subject name")
+    public void checkExistedSubjectNameErrorMessage() {
+        logger.info("Check error message of existed subject name");
+        assertThat(subjectPopup.getExistedSubjectNameErrorMessage())
+                .withFailMessage("Error message of existed subject name is incorrect")
+                .isEqualTo(ErrorMessagesProvider.getExistedSubjectNameErrMessage());
+    }
+
+    @Then("Check matched items are appeared below the Search line")
+    public void checkMatchedItemsAreAppearedBelowTheSearchLine() {
+        assertThat(subjectPopup.checkMatchedItemsAreAppearedBelowTheSearchLine())
+                .withFailMessage("Matched items are not appeared")
+                .isTrue();
+    }
+
+    @Then("Check selected items are shown with the 'x' icon")
+    public void checkSelectedItemIsShownWithTheXIcon() {
+        assertThat(subjectPopup.checkSelectedItemsAreShownWithTheXIcon())
+                .withFailMessage("Selected item is not shown")
+                .isTrue();
+    }
+
+    @Then("Check selected items are deleted from drop-list fragment")
+    public void checkSelectedItemIsDeletedFromDropListFragment() {
+        logger.info("Check selected items are deleted from drop-list fragment");
+        assertThat(subjectPopup.checkThereAreNoSelectedItems())
+                .withFailMessage("Selected item is not deleted")
+                .isTrue();
+    }
+
+    @Then("Check there is no selected item")
+    public void checkThereIsNoSelectedItem() {
+        logger.info("Check there is no selected item");
+        assertThat(subjectPopup.checkThereAreNoSelectedItems())
+                .withFailMessage("There is selected item")
+                .isTrue();
+    }
+
+    @Then("Check the search line placeholder")
+    public void checkTheSearchLinePlaceholder() {
+        logger.info("Check the search line placeholder");
+        assertThat(subjectPopup.getTheSearchLinePlaceholder())
+                .withFailMessage("There is selected item")
+                .isEqualTo("Select teachers");
```

Comment:
I wonder if it is correct to hard code "Select teachers" here, you can add it to property file and read from there

Reply:
Resolved